doc/service_reference_card.md

```diff
@@ -0,0 +1,56 @@
+# Service Reference Card Template
+
+* Daemons running
+  * httpd
+  * crond
+  * atd
+  * apeldbloader-cloud
+
+* Init commands (services started on container creation)
+  * service httpd start
+  * service crond start
+  * service atd start
+  * service apeldbloader-cloud start
+
+* Configuration files location
+  * /etc/httpd/conf.d/apel_rest_api.conf - forces SSL
+  * /etc/httpd/conf.d/ssl.conf - configures SSL
+  * /etc/apel/cloudloader.cfg - configuration for the loader
+  * /etc/apel/cloudsummariser.cfg - configuration for the summariser
+
+* Logfile locations
+  * /var/log/cloud/loader.log - log file for the loader
+  * /var/log/cloud/summariser.log - log file for the summariser
+
+* Open ports
```

Comment:
I have changed it to 

* Open ports
  * 80 - all traffic to this port is forwarded to port 443 by the Apache server.
  * 443 - the Apache server forwards (HTTPS) traffic to the APEL server, which returns a Django view for recognised URL patterns.
  * 3306 - used by the APEL server service and the MySQL service to communicate with each other.



Shall I change in admin.md as well?